tests/sentry/api/endpoints/test_organization_metric_data.py

```diff
@@ -506,31 +506,17 @@ def count_sessions(project_id: Optional[int]) -> int:
         assert count_sessions(project_id=self.project2.id) == 1
 
     def test_max_and_min_on_distributions(self):
-        # Record some strings
-        org_id = self.organization.id
-        k_transaction = perf_indexer_record(org_id, "transaction")
-        v_foo = perf_indexer_record(org_id, "/foo")
-        v_bar = perf_indexer_record(org_id, "/bar")
-        v_baz = perf_indexer_record(org_id, "/baz")
-
-        self._send_buckets(
-            [
-                {
-                    "org_id": org_id,
-                    "project_id": self.project.id,
-                    "metric_id": self.transaction_lcp_metric,
-                    "timestamp": int(time.time()),
-                    "tags": {
-                        k_transaction: v_transaction,
-                    },
-                    "type": "d",
-                    "value": [123.4 * count],
-                    "retention_days": 90,
-                }
-                for v_transaction, count in ((v_foo, 1), (v_bar, 3), (v_baz, 2))
-            ],
-            entity="metrics_distributions",
-        )
+        for v_transaction, count in (("/foo", 1), ("/bar", 3), ("/baz", 2)):
```

Comment:
it should now be possible to remove `broken_under_tags_values_as_strings` from this file and still have the tests pass